api/src/utilities/passport-validator-utilities.ts

```diff
@@ -6,14 +6,14 @@ import { HttpException, HttpStatus } from '@nestjs/common';
  * @param failedLoginAttemptsCount the number of times the user failed to log in (stored in db)
  * @param maxAttempts the maximum number of attempts before user is considered locked out (env variable)
  *
- * @returns throws error if user is already locked out
+ * @returns updated value of failed login attempts
  */
-export function isUserLockedOut(
+export function checkUserLockout(
```

Comment:
Really struggled with naming this function since it's handling both the erroring case and the reset failed attempts case in one. I considered separating them out but it seemed most of logic would be duplicated... any thoughts?